components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -118,15 +115,6 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterDictionaryPref(kBraveWalletLastTransactionSentTimeDict);
   registry->RegisterTimePref(kBraveWalletLastDiscoveredAssetsAt, base::Time());
 
-  // TODO(djandries): remove the following prefs at some point,
-  //                  since they're now maintained in local state
-  p3a_utils::RegisterFeatureUsagePrefs(registry, kBraveWalletP3AFirstUnlockTime,
-                                       kBraveWalletP3ALastUnlockTime,
-                                       kBraveWalletP3AUsedSecondDay, nullptr);
-  registry->RegisterTimePref(kBraveWalletLastUnlockTime, base::Time());
-  registry->RegisterTimePref(kBraveWalletP3ALastReportTime, base::Time());
-  registry->RegisterTimePref(kBraveWalletP3AFirstReportTime, base::Time());
-  registry->RegisterListPref(kBraveWalletP3AWeeklyStorage);
```

Comment:
We should not remove it directly like this. It has to be put them to places like `MigrateObsoleteProfilePrefs` and clear it, otherwise these prefs would persist in user disk forever. 
And after certain amount of time like 1 year, at that point basically most of the users have run this migration code then we can remove the pref and the migration code together.

Reply:
We have had profile -> local migration since November here: https://github.com/brave/brave-core/blob/3d311c2a5715fec3e02093d1814dba3d6a389f89/components/brave_wallet/browser/brave_wallet_p3a.cc#L333

i assume what you mean is that we should add pref clearing calls to the migration function before wiping it out. is that right?

Reply:
That's right, the main purpose is to clear these prefs.

Reply:
seems like you already clear them in `BraveWalletP3A::MigrateUsageProfilePrefsToLocalState`

Reply:
actually it is only `kBraveWalletP3AUsedSecondDay` and `kBraveWalletP3AWeeklyStorage`

Reply:
we also migrated these

https://github.com/brave/brave-core/blob/3d311c2a5715fec3e02093d1814dba3d6a389f89/components/brave_wallet/browser/brave_wallet_p3a.cc#L54-L57

via 

https://github.com/brave/brave-core/blob/3d311c2a5715fec3e02093d1814dba3d6a389f89/components/brave_wallet/browser/brave_wallet_p3a.cc#L334-L342